src/locale/en/help.json

```diff
@@ -75,6 +73,15 @@
           "A validator can update their commission rates as and when they please, and such changes will have an impact on your profitability. Be sure to monitor your nominations on this dashboard to keep updated on their commission rates."
         ]
       ],
+      "decentralizationAnalyticsPeriod": [
+        "Decentralization Analytics Period",
+        [
+          "The decentralization of your nomination is calculated analysing the rewards payed out in the last 30 days.",
+          "Depending on the Network this will translate into a number of Eras.",
+          "30 days provides enough information for the aggregation to be meaningful.",
+          "30 days is also short enough for changes in nominations to start impacting this analysis after a few days."
```

Comment:
```suggestion
          "The decentralization of your nomination is calculated by analysing the rewards payed out in the last 30 days. Depending on the network, this will translate into a number of Eras.",
          "30 days provides enough information for the aggregation to be meaningful, and is also short enough for changes in nominations to start impacting this analysis after a few days."
```